app/views/default/index.html.erb

```diff
@@ -19,19 +19,19 @@
     <div class="container">
         <div class="row">
             <div class="col-3 bubbles rounded text-center">
-                <h2>Learn about systemic racism</h2>
-                <%= link_to image_tag('button_timeline.png', class: "img-fluid pt-5", alt: "timeline"), :timeline %>
+                <h2><%= link_to "Learn about systemic racism", :timeline%></h2>
+                <%= link_to image_tag('button_timeline.png', class: "img-fluid pt-5 d-none d-md-block", alt: "timeline"), :timeline %>
             </div>
```

Comment:
link the headers and then apply classes to hide the images on narrow width screens